nncf/quantization/quantize_functions.py

```diff
@@ -162,17 +172,14 @@ def get_scale_zp_from_input_low_input_high(level_low, level_high, input_low, inp
     level_high = level_high.to(y_zero_point.device)
     y_zero_point = torch.min(torch.max(level_low, y_zero_point.to(type_)), level_high)
 
-    per_channel_idx = torch.where(torch.tensor(y_scale.shape) != 1)[0]
-    if len(per_channel_idx) == 0:
-        per_channel_idx = 0
-    elif len(per_channel_idx) == 1:
-        per_channel_idx = per_channel_idx[0].item()
-    else:
-        raise RuntimeException('Scale parameter can not be squeezed to one dimensional array')
+    _, per_channel_idx = get_channel_count_and_dim_idx(list(y_scale.shape))
 
-    y_scale = torch.squeeze(y_scale)
-    y_zero_point = torch.squeeze(y_zero_point)
-    return y_scale, y_zero_point, per_channel_idx
+    if len(y_scale.shape) > 1:
+        y_scale = torch.squeeze(y_scale)
+        y_zero_point = torch.squeeze(y_zero_point)
+        return y_scale, y_zero_point, per_channel_idx
+    else:
```

Comment:
I think even without this branch of `if` it should work fine but there will be less code.

Reply:
There may be a subtle difference between `torch.Size([])` and `torch.Size([1])` in further processing, I added this code to avoid future potential problems.